realtime/flood/peta_jakarta_api.py

```diff
@@ -17,7 +17,8 @@ class PetaJakartaAPI(object):
 
     @classmethod
     def get_aggregate_report(cls, duration, level):
-        rest_point = 'https://rem.petajakarta.org/banjir/data/api/v2/rem/flooded'
+        rest_point = \
```

Comment:
Which is better, using backslash or brackets for case like this?

In my knowledge, backslash has a risk to introduce an error since it's not easy to see a line continuation.